.github/workflows/code-quality.yml

```diff
@@ -7,6 +7,20 @@ on:
       - "*.latest"
   pull_request:
   workflow_dispatch:
+    inputs:
+      dbt_adapters_branch:
+        inputs:
```

Comment:
You must really want these to be inputs.

Reply:
You have no idea